### PR TITLE
Remove Parquet amalgamation

### PR DESCRIPTION
**Pull request by majetideepak**
Keep ParquetDecodeUtils and ByteBuffer needed by the BitPackDecoderBenchmark.cpp

### CODE REVIEW

**Comment by pedroerp** [comment]
Nice, thanks for quickly cleaning this up! Once the CI signal is ready (and green) I'll merge it

**Comment by majetideepak** [comment]
Once we resolve https://github.com/facebookincubator/velox/pull/6519, this should be good to go.

**Comment by majetideepak** [comment]
@mbasmanova Since Pedro is taking time off, can you please help review and merge this? Thanks.

**Review by mbasmanova — COMMENTED** [review]
Perhaps, update velox/duckdb/README.md

**Review by mbasmanova — COMMENTED** [review]
Maybe move the remaining pieced into XxxBenchmark.cpp?

**Comment by majetideepak** [comment]
@mbasmanova addressed your comments. Thanks!

File: velox/dwio/common/tests/BitPackDecoderBenchmark.cpp [hunk 1]
```diff
@@ -38,6 +37,187 @@ using RowSet = folly::Range<const facebook::velox::vector_size_t*>;
 
 static const uint64_t kNumValues = 1024768 * 8;
 
+namespace duckdb {
+
+class ByteBuffer { // on to the 10 thousandth impl
```
Comment (mbasmanova) [inline velox/dwio/common/tests/BitPackDecoderBenchmark.cpp:42]: nit: consider fixing coding style

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Comment by mbasmanova** [comment]
@majetideepak Deepak, I'm getting an error:

```
fbcode/velox/external/duckdb/duckdb-5.cpp:16774:10: fatal error: 'parquet-extension.hpp' file not found
#include "parquet-extension.hpp"
         ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

**Comment by mbasmanova** [comment]
@majetideepak I see that I need to remove BUILD_PARQUET_EXTENSION.

**Pull request closed without merging**